guides/source/getting_started.md

```diff
@@ -7,1150 +7,1554 @@ This guide covers getting up and running with Ruby on Rails.
-  things in a web application, and defaults to this set of conventions, rather than
-  require that you specify minutiae through endless configuration files.
-
-Creating a New Rails Project
-----------------------------
-
-TIP: You can create new Rails apps with a preconfigured Dev Container development environment. This
-is the fastest way to get started with Rails.
-For instructions see [Getting Started with Dev Containers](getting_started_with_devcontainer.html)
-
-The best way to read this guide is to follow it step by step. All steps are
-essential to run this example application and no additional code or steps are
-needed.
-
-By following along with this guide, you'll create a Rails project called
-`blog`, a (very) simple weblog. Before you can start building the application,
-you need to make sure that you have Rails itself installed.
-
-NOTE: The examples below use `$` to represent your terminal prompt in a UNIX-like OS,
-though it may have been customized to appear differently. If you are using Windows,
-your prompt will look something like `C:\source_code>`.
-
-### Installing Rails
-
-Before you install Rails, you should check to make sure that your system has the
-proper prerequisites installed. These include:
-
-* Ruby
-* SQLite3
-
-#### Installing Ruby
-
-Open up a command line prompt. On macOS open Terminal.app; on Windows choose
-"Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a
-dollar sign `$` should be run in the command line. Verify that you have a
-current version of Ruby installed:
-
-```bash
-$ ruby --version
-ruby 3.2.0
-```
-
-Rails requires Ruby version 3.2.0 or later. It is preferred to use the latest Ruby version.
-If the version number returned is less than that number (such as 2.3.7, or 1.8.7),
-you'll need to install a fresh copy of Ruby.
+- **Don't Repeat Yourself:** DRY is a principle of software development which states that "Every piece of knowledge must have a single, unambiguous, authoritative representation within a system". By not writing the same information over and over again, our code is more maintainable, more extensible, and less buggy.
+- **Convention Over Configuration:** Rails has opinions about the best way to do many things in a web application, and defaults to this set of conventions, rather than require that you define them yourself through endless configuration files.
 
-To install Rails on Windows, you'll first need to install [Ruby Installer](https://rubyinstaller.org/).
+Creating a New Rails App
+------------------------
 
-For more installation methods for most Operating Systems take a look at
-[ruby-lang.org](https://www.ruby-lang.org/en/documentation/installation/).
+We're going to build a project called `store` - a simple e-commerce app that demonstrates several of Rails' built-in features.
 
-#### Installing SQLite3
+TIP: Any commands prefaced with a dollar sign `$` should be run in the terminal.
```

Comment:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/f8f07eab-9306-4b28-940e-be7a49b2fe02">

Taking this copy, making it a TIP and placing it right after "Creating a New Rails App" is such a GREAT UX.

It's very easy for us to remember this but it makes the guide accessible to people at the "Hello World" level. Love it.